### DISCUSSION TITLE
Welcome to DAFoam Discussions!

### DISCUSSION THREAD

**Discussion by friedenhe** [mdolab/dafoam#67] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Karyan07** [comment]
I am a local user in this Linux system (CentOS7) and I do not have administrator privileges, while using OpenFOAM and DAFoam,
As a local user, when I run docker and when I try to run the Airfoil code given in the tutorial, the following error appears when running the preProcessing.sh script.

"./preProcessing.sh: line 11: logMeshGeneration.txt: Permission denied
./preProcessing.sh: line 12: logMeshGeneration.txt: Permission denied
./preProcessing.sh: line 13: logMeshGeneration.txt: Permission denied
./preProcessing.sh: line 14: logMeshGeneration.txt: Permission denied
./preProcessing.sh: line 15: logMeshGeneration.txt: Permission denied
"
and it says
"
Generating mesh.. Done!
cp: cannot create directory '0': Permission denied
"
LogOpt.txt not getting generated and the code doesnt seem to run.

Please tell me a way to fix this.
